config/settings.yml

```diff
@@ -5,7 +5,7 @@ purl_conn_timeout: 2
 purl_feedback_email: ''
 valid_purl_url: <%= /https?:\/\/purl\.stanford\.edu\/*/ %>
 stacks_url: 'https://stacks.stanford.edu'
-iiif_stacks_url: 'https://stacks.stanford.edu'
+iiif_stacks_url: 'https://stacks.stanford.edu' # unused???
```

Comment:
Yeah, it doesn't look like it's used anymore (I'm guessing it did at one point before we swapped out the image viewer).

I say kill it.